tonic-flatbuffers-build/src/builder.rs

```diff
@@ -289,7 +289,22 @@ impl Builder {
 
         // Convert all flatbuffer RPC definitions into protobuf RPC definitions
         for file in fbs {
-            let file = std::fs::read_to_string(file)?.replace(['\r', '\n'], "");
+            let file = std::fs::read_to_string(file)?;
+
+            // cleanup comments above rpc definition using ///
+            let file = file
+                .lines()
+                .filter_map(|line| {
+                    if !line.trim_start().starts_with("///") {
+                        Some(line.to_string())
+                    } else {
```

Comment:
Because `//` still produces a valid comment (just not for documentation), I think you should filter out lines that start with `//` instead